src/appgateway.tf

```diff
@@ -104,10 +114,11 @@ module "app_gw" {
   listeners = {
 
     api = {
-      protocol         = "Https"
-      host             = format("api.%s.%s", var.dns_zone_prefix, var.external_domain)
-      port             = 443
-      ssl_profile_name = format("%s-ssl-profile", local.project)
+      protocol           = "Https"
+      host               = format("api.%s.%s", var.dns_zone_prefix, var.external_domain)
+      port               = 443
+      ssl_profile_name   = format("%s-ssl-profile", local.project)
+      firewall_policy_id = null
 
       certificate = {
```

Comment:
also use replace instead of trimsuffix

https://github.com/pagopa/io-infra/blob/main/src/core/appgateway.tf#L168

Reply:
thanks, also for management_platform and portal_platform

Reply:
done